doc/source/ray-air/examples/tfx_tabular_train_to_serve.ipynb

```diff
@@ -6,6 +6,8 @@
     "id": "VaFMt6AIhYbK"
    },
    "source": [
+    "# Tabular data training and serving using Ray Air\n",
```

Comment:
The index page only renders with the first level 1 header in the notebook. Currently it shows "set up ray"

Reply:
nice :)